<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plc_monitorClass</class>
 <widget class="QMainWindow" name="plc_monitorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>866</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>plc_monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="PlctabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="setting">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="control_register_on">
          <property name="text">
           <string>触发:开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="control_register_off">
          <property name="text">
           <string>触发:关</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="trigger_coder">
          <property name="text">
           <string>编码器读数重置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="trigger_marking">
          <property name="text">
           <string>打标</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearCableMil">
          <property name="text">
           <string>清除里程</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="trigger_switch_on">
          <property name="text">
           <string>电源:开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="trigger_switch_off">
          <property name="text">
           <string>电源:关</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="light_control_on">
          <property name="text">
           <string>告警灯控制:开</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="light_control_off">
          <property name="text">
           <string>告警灯控制:关</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ocr_mil">
          <property name="text">
           <string>字符间隔告警</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ocr_others">
          <property name="text">
           <string>字符其他告警</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="surface_ln">
          <property name="text">
           <string>凹凸告警</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="surface_dia">
          <property name="text">
           <string>直径告警</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="surface_sur">
          <property name="text">
           <string>表面告警</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="comboBox_buzzer_mode">
            <property name="currentText">
             <string/>
            </property>
            <property name="maxCount">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buzzer_mode">
            <property name="text">
             <string>蜂鸣器模式</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QSpinBox" name="tbshort_buzzing_time">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="short_buzzing_time">
            <property name="text">
             <string>设置短间隔蜂鸣器时间</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QSpinBox" name="tblong_buzzing_time">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="long_buzzing_time">
            <property name="text">
             <string>设置长间隔蜂鸣器时间</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QComboBox" name="comboBox_short_buzzing_interval">
            <property name="currentText">
             <string/>
            </property>
            <property name="maxCount">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="short_buzzing_interval">
            <property name="text">
             <string>设置短间隔蜂鸣器间隔时间</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="CableLength">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="setCableLength">
            <property name="text">
             <string>设置里程数</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSpinBox" name="CamTriggerInterval">
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="setCamTriggerInterval">
            <property name="text">
             <string>辅助相机间隔</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Connect">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>570</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="data">
       <attribute name="title">
        <string>实时数据</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="plc_monitor.qrc"/>
 </resources>
 <connections/>
</ui>
